lib/active_record_view_model/update_data.rb

```diff
@@ -171,49 +171,88 @@ def self.empty_update_for(viewmodel)
       self.new(viewmodel.class, viewmodel.id, {}, [])
     end
 
+    # Normalised handler for looking at the UpdateData for an association
+    def visit_association(name, value, empty:, map:, reduce:)
```

Comment:
Please rename these. `reduce_association`? `process_association`? Just not `visit_` please.